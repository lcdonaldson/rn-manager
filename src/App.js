
import React, { Component } from "react";
import { Platform, StyleSheet, Text, View } from "react-native";
// import Button from '../src/components/Button';
import {
    Container,
    Header,
    Left,
    Body,
    Right,
    Button,
    Icon,
    Title
} from "native-base";

export default class App extends Component {
    render() {
        return (
            <Container>
                <Header>
                    <Left>
                        <Button transparent>
                            <Icon name="menu" />
                        </Button>
                    </Left>
                    <Body>
                        <Title>Manager</Title>
                    </Body>
                    <Right>
                        <Button transparent>
                            <Icon name="search" />
                        </Button>
                    </Right>
                </Header>

                <View style={styles.container}>
                    <Text style={styles.welcome}>Welcome to React Native!</Text>
                </View>
            </Container>
        );
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: "center",
        alignItems: "center",
        backgroundColor: "#F5FCFF"
    },
    welcome: {
        fontSize: 20,
        textAlign: "center",
        marginTop: 10
    },
    instructions: {
        textAlign: "center",
        color: "#333333",
        marginBottom: 5
    }
});
