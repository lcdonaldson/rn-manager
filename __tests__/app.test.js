import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/App';
