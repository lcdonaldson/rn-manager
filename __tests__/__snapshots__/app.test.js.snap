// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "teal",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "borderBottomWidth": 0,
          "borderColor": "#ccc",
          "borderLeftWidth": 0,
          "borderRadius": 2,
          "borderRightWidth": 0,
          "borderTopWidth": 0,
          "elevation": null,
          "flexWrap": "nowrap",
          "marginBottom": 5,
          "marginLeft": 2,
          "marginRight": 2,
          "marginTop": 5,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
        },
        Object {
          "backgroundColor": "transparent",
          "borderColor": "teal",
          "marginBottom": 250,
          "maxHeight": 150,
        },
      ]
    }
    transparent={true}
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
            "fontSize": 40,
            "fontWeight": "400",
            "textAlign": "center",
          }
        }
      >
        Manager
      </Text>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "marginTop": 20,
            "maxHeight": 40,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#00b386",
              "borderRadius": 20,
              "marginHorizontal": 10,
              "minHeight": 40,
              "opacity": 1,
              "padding": 5,
              "width": 120,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            Login
          </Text>
        </View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#00b386",
              "borderRadius": 20,
              "marginHorizontal": 10,
              "minHeight": 40,
              "opacity": 1,
              "padding": 5,
              "width": 120,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "fontSize": 18,
                "fontWeight": "bold",
              }
            }
          >
            Sign Up
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
